databricks/koalas/indexes.py

```diff
@@ -123,6 +123,29 @@ def size(self) -> int:
         """
         return len(self._kdf)  # type: ignore
 
+    @property
+    def shape(self) -> tuple:
+        """
+        Return a tuple of the shape of the underlying data.
+
+        Examples
+        --------
+        >>> idx = ks.Index(['a', 'b', 'c'])
+        >>> idx
+        Index(['a', 'b', 'c'], dtype='object')
+
+        >>> midx = ks.MultiIndex.from_tuples([('a', 'x'), ('b', 'y'), ('c', 'z')])
+        >>> midx  # doctest: +SKIP
+        MultiIndex([('a', 'x'),
+                    ('b', 'y'),
+                    ('c', 'z')],
+                   )
+
+        >>> midx.shape
+        (3,)
+        """
+        return len(self._kdf),
```

Comment:
i'm not pretty sure, but maybe `return (len(self._kdf),)` ? 😃 

now pandas works like below. 

```python
>>> pidx = pd.Index(['a', 'b' ,'c'])
>>> pidx.shape
(3,)
```